### DISCUSSION TITLE
Add 2nd OS controller??

### DISCUSSION THREAD

**Discussion by PerfectBake420** [vinteo/hass-opensprinkler#177] [Q&A] [answered]
Hello. I am wondering if it is possible to integrate 2 separate OpenSprinklers? I have 1 integrated with the 72 stations it can hold. I need to add another 72 stations. Can this be done?

**Comment by vinteo** [comment, marked as answer]
Yes it should work.

More detailed instructions here: https://github.com/vinteo/hass-opensprinkler/discussions/143#discussioncomment-275495
